src/main/java/com/google/api/codegen/configgen/viewmodel/FieldNamePatternView.java

```diff
@@ -16,10 +16,13 @@
 
 import com.google.auto.value.AutoValue;
 
+/** Groups a field name of the request types with an entity name of the collections. */
 @AutoValue
 public abstract class FieldNamePatternView {
+  /** The path of the field. */
```

Comment:
Ah, I'm not sure I fully recognized this originally. We call this concept "pathTemplate" other places in the codebase; can we rename it here?

Also, since "path" is ambiguous, perhaps a description like "the template representing the structure of the resource name encoded in the field"?